src/devcenter/azext_devcenter/manual/custom.py

```diff
@@ -146,71 +146,30 @@ def devcenter_network_connection_list_health_detail(
     )
 
 
-def devcenter_pool_create(
-    client,
-    resource_group_name,
-    project_name,
-    pool_name,
-    location,
-    dev_box_definition_name,
-    network_connection_name,
-    local_administrator,
-    license_type,
-    tags=None,
-    no_wait=False,
-):
+def devcenter_pool_create(client,
+                          resource_group_name,
+                          project_name,
+                          pool_name,
+                          location,
+                          dev_box_definition_name,
+                          network_connection_name,
+                          local_administrator,
+                          tags=None,
+                          no_wait=False):
     body = {}
     if tags is not None:
-        body["tags"] = tags
-    body["location"] = location
-    body["dev_box_definition_name"] = dev_box_definition_name
-    body["network_connection_name"] = network_connection_name
-    body["license_type"] = license_type
-    body["local_administrator"] = local_administrator
-    return sdk_no_wait(
-        no_wait,
-        client.begin_create_or_update,
-        resource_group_name=resource_group_name,
-        project_name=project_name,
-        pool_name=pool_name,
-        body=body,
-    )
-
-
-def devcenter_pool_update(
```

Comment:
Not sure why this in this file, there is no customization to the update pool command so deleting this function so that the CLI will reference the one in generated/custom.py